src/app/color/color-palette.demo.scss

```diff
@@ -1,6 +1,8 @@
-// Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
-// This software is released under MIT license.
-// The full license information can be found in LICENSE in the root directory of this project.
+/*!
```

Comment:
I got to thinking, is it even necessary to declare the year? Angular doesn't https://github.com/angular/angular/blob/master/packages/common/src/common.ts.

It links to a page on their website that contains the date. This is an annoying housekeeping task that would be nice to remove, but I suspect we'd have to talk to legal.

Reply:
Yes, I believe the copyright text for files was given to us. Perhaps it s worth asking if the year is needed?